sql/sql_class.h

```diff
@@ -112,6 +112,9 @@
 #include "thr_lock.h"
 #include "violite.h"
 
+#include <boost/property_tree/ptree.hpp>
+using boost::property_tree::ptree;
```

Comment:
`using` directives should not be used in header files.
I'd rather define a new type for this
```suggestion
using metadata_ptree = boost::property_tree::ptree;
```